AllReadyApp/Web-App/AllReady/web.config

```diff
@@ -1,9 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <system.webServer>
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\AllReady.exe" arguments="" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
```

Comment:
@mgmccarthy I've had to make this change locally to get it building and running. I think this was originally the setting until the RTM work. Do we know if this will still build / run in AppVeyor if this is committed?